src/libsystemd/sd-netlink/sd-netlink.c

```diff
@@ -227,6 +227,41 @@ int sd_netlink_send(sd_netlink *nl,
         return 1;
 }
 
+int sd_netlink_sendv(sd_netlink *nl,
+                     sd_netlink_message *messages[],
+                     size_t msgcount,
+                     uint32_t **serial) {
```

Comment:
we usually prefix return params `ret_`, please do so here too, i.e. `serial`→ `ret_serial` to make it easier to see what is output.

(if there's only a single return param and and it's the primary purpose of the function to return it, you may also call the return param just `ret`, i.e without any further suffix)